src/doc/tarpl/meet-safe-and-unsafe.md

```diff
@@ -0,0 +1,98 @@
+% Meet Safe and Unsafe
+
+Programmers in safe "high-level" languages face a fundamental dilemma. On one
+hand, it would be *really* great to just say what you want and not worry about
+how it's done. On the other hand, that can lead to some *really* poor
```

Comment:
Warning: Super nitpicky opinions about writing style follows.

 > Rather, very, little, pretty -- these are the leeches that infest the pond of prose, sucking the blood of words. 

-- William Strunk

I think you can safely throw "really" into this group as well. The use of "really" in this line is unnecessary and distracting and should be removed. The use in the previous line seems okay to me, though.

Also, you are *way* overusing italics. I struggled to get through the first 10 paragraphs of this chapter, and had to constantly fight the urge to stop reading, fork the book and rip out most of the italics so I could properly read it.